index.js

```
@@ -55,10 +70,11 @@ const replaceMdIgnoringCode = (mdString, replaceFn) => {
 // DATA STRUCTURES
 
 class PumlLinks extends Map {
-    constructor({ pumlPaths, shouldShortenLinks, pumlServerUrl }) {
+    constructor({ pumlPaths, shouldShortenLinks, pumlServerUrl, polrUrl }) {
         super();
         this.shouldShortenLinks = shouldShortenLinks
```

can we throw an error if both polrUrl and shouldShortenLinks are true?

---

changed code so that if both options are specified, error is logged to console and program terminates.